### DISCUSSION TITLE
How to used pretrain models for single channel images

### DISCUSSION THREAD

**Discussion by sabernn** [qubvel/segmentation_models.pytorch#552] [Q&A]
Hi,
Thank you for the great repo. I've been trying to run the model with pre-trained weights for grayscale images but it keeps giving me errors, even when I set in_channels=1 in model instantiation. Is there a way to fix that?

Saber

**Comment by qubvel** [comment]
For grayscale images you have to set in_channels=1.
Anyway it is impossible to figure out what's going wrong without code reproducing the bug and traceback

**Reply by sabernn** [reply]
I already did that. But it seems the pre-trained backbones are only trained for RGB images.

**Reply by nquaz** [reply]
just to confirm, my code works for greyscale on pre-trained backbone. could you send your code?
